[2025-06-27T06:23:54.383+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T06:23:54.458+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0_dev.start_task scheduled__2025-03-10T02:10:00+00:00 [queued]>
[2025-06-27T06:23:54.526+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0_dev.start_task scheduled__2025-03-10T02:10:00+00:00 [queued]>
[2025-06-27T06:23:54.537+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T06:23:54.794+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): start_task> on 2025-03-10 02:10:00+00:00
[2025-06-27T06:23:54.845+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1349) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T06:23:54.851+0000] {standard_task_runner.py:72} INFO - Started process 1380 to run task
[2025-06-27T06:23:54.888+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_v1.0_dev', 'start_task', 'scheduled__2025-03-10T02:10:00+00:00', '--job-id', '330', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmp_kw6xjba']
[2025-06-27T06:23:54.921+0000] {standard_task_runner.py:105} INFO - Job 330: Subtask start_task
[2025-06-27T06:23:55.255+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_v1.0_dev.start_task scheduled__2025-03-10T02:10:00+00:00 [running]> on host 4e2c61145aac
[2025-06-27T06:23:55.739+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_v1.0_dev' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T02:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T02:10:00+00:00'
[2025-06-27T06:23:55.750+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T06:23:55.796+0000] {logging_mixin.py:190} INFO - start
[2025-06-27T06:23:55.802+0000] {logging_mixin.py:190} INFO - ds: 2025-03-10
[2025-06-27T06:23:55.805+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-06-27T06:23:55.851+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T06:23:55.853+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_v1.0_dev, task_id=start_task, run_id=scheduled__2025-03-10T02:10:00+00:00, execution_date=20250310T021000, start_date=20250627T062354, end_date=20250627T062355
[2025-06-27T06:23:55.910+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T06:23:56.085+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
